Documentation/AutoRetry.md

```diff
@@ -0,0 +1,39 @@
+# Auto retry
+----
+
+## Auto refresh token
+
+Compose with this property, you can inject it when you want :
```

Comment:
Using this property, you can tell your code how to renew the token. Typically, this closure will contain code to perform the "RefreshToken" request, parse the response, store the new token in your NetworkStack, and emit a `Observable.just()` to tell when it's done.